### PR TITLE
Build info

### PR DESCRIPTION
**Pull request by izzyblues**
Implemented build-info and ping endpoints by using the brand new go shared library

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,9 +11,17 @@ RUN apk add --update bash \
   && cd $GOPATH/src/${REPO_PATH} \
   && go get -t ./... \
   && go test ./... \
-  && go build \
+  && LDFLAGS="$(${GOPATH}/src/github.com/Financial-Times/service-status-go/buildinfo/ldFlags.sh)" \
+  && go build -ldflags="${LDFLAGS}" \
   && mv content-preview /content-preview-app \
   && apk del go git bzr \
   && rm -rf $GOPATH /var/cache/apk/*
```
Comment (kghatton) [inline Dockerfile:18]: In Java services we typically rm the source after running Maven, so you might want to remove `.git` afterwards as well.

File: Dockerfile [hunk 2]
```diff
@@ -12,9 +11,17 @@ RUN apk add --update bash \
...
 
-CMD exec /content-preview-app --app-port $APP_PORT --source-app-auth $SOURCE_APP_AURTH --source-app-uri $SOURCE_APP_URI  --transform-app-uri $TRANSFORM_APP_URI --transform-app-host-header $TRANSFORM_APP_HOST_HEADER --source-app-health-uri $SOURCE_APP_HEALTH_URI --transform-app-health-uri $TRANSFORM_APP_HEALTH_URI
+CMD exec /content-preview-app \
+		--app-port $APP_PORT \
+		--source-app-auth $SOURCE_APP_AURTH \
```
Comment (ghost) [inline Dockerfile]: typo here. (although  it was already there before) "AURTH"

File: Dockerfile (hunk 1 above)
Comment (izzyblues) [inline Dockerfile:18]: I think that the .git folder is deleted by this line:
`rm -rf $GOPATH /var/cache/apk/`

File: Dockerfile (hunk 2 above)
Comment (izzyblues) [inline Dockerfile]: Fixed typo

**Comment by JuliaFernee** [comment]
+1 It works

**Pull request merged**